common/djangoapps/tedix_ro/views.py

```diff
@@ -197,8 +199,16 @@ def manage_courses(request):
                         send_mail(subject, txt_message, from_address, [parent.user.email], html_message=html_message)
 
                 if form.cleaned_data['send_sms']:
-                    # sending sms logic to be here
-                    pass
+                    parent = student.parents.first()
```

Comment:
Do we want to check this parent for existence before?

Reply:
student cannot register without parent.